src/rpc/protocol.cpp

```diff
@@ -79,8 +79,6 @@ boost::filesystem::path GetAuthCookieFile()
 boost::filesystem::path GetMainchainAuthCookieFile()
 {
     boost::filesystem::path path(GetArg("-mainchainrpccookiefile", COOKIEAUTH_FILE));
-    if (!path.is_complete() && BaseParams().DataDir() == CHAINPARAMS_ELEMENTS) path = "testnet3" / path;
-    if (!path.is_complete() && BaseParams().DataDir() == CHAINPARAMS_REGTEST) path = "regtest" / path;
```

Comment:
wouldn't this break connecting via rpc cookie auth?

Reply:
They should just put it in elementsregtest, no?

Reply:
this is specifically looking for the `bitcoind` cookie, not elementsd.